plugins/woocommerce/src/Blocks/BlockTypes/AddToCartWithOptions.php

```diff
@@ -237,8 +235,9 @@ protected function render( $attributes, $content, $block ) {
 			remove_filter( 'render_block_context', array( $this, 'set_is_descendant_of_add_to_cart_with_options_context' ) );
 
 			$form_html = sprintf(
-				'<form %1$s>%2$s%3$s%4$s</form>',
```

Comment:
To me, I still prefer defining the context data as the wrapper attributes for the readability. Using `wp_interactivity_data_wp_context` this way works, but it make this hard to line LOC harder to read, which in a way, goes against the purpose of refactoring imho. Curious to see what others think :D

Reply:
Perhaps we could take this opportunity to introduce the additional helper that doesn't include the attribute name and see how that one feels.

Reply:
> the additional helper that doesn't include the attribute name

Yes, I was thinking about extracting the current help into a new one such as `wp_interactivity_prepare_data_wp_context` and refactor the `wp_interactivity_data_wp_context` using the new helper.